### ISSUE TITLE
[Bug] Crashing

### ISSUE THREAD

**Issue by NotMihir** [BiscuitDevelopment/SkyblockAddons#488]
Please note that we do not provide support for clients other than Forge.
We will not fix compatibility issues with other clients.

**Describe the bug**
Just Crashes the game on launch

**To Reproduce**
Steps to reproduce the behavior (how did you get this bug to occur):
1. Just Run mc

**Crash Report**
If your game crashed, please paste your crash report below.

```text
---- Minecraft Crash Report ----

WARNING: coremods are present:
  SkyblockAddonsLoadingPlugin (SkyblockAddons-1.6.0-beta.7-for-MC-1.8.9.jar)
Contact their authors BEFORE contacting forge

// Uh... Did I do that?

Time: 1/22/21 4:30 PM
Description: There was a severe problem during mod loading that has caused the game to fail

net.minecraftforge.fml.common.LoaderException: java.lang.NoSuchMethodError: com.google.gson.JsonParser.parseString(Ljava/lang/String;)Lcom/google/gson/JsonElement;
	at net.minecraftforge.fml.common.LoadController.transition(LoadController.java:162)
	at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:744)
	at net.minecraftforge.fml.client.FMLClientHandler.finishMinecraftLoading(FMLClientHandler.java:310)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:495)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:329)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
Caused by: java.lang.NoSuchMethodError: com.google.gson.JsonParser.parseString(Ljava/lang/String;)Lcom/google/gson/JsonElement;
	at codes.biscuit.skyblockaddons.utils.Utils.loadLanguageFile(Utils.java:662)
	at codes.biscuit.skyblockaddons.utils.Utils.loadLanguageFile(Utils.java:645)
	at codes.biscuit.skyblockaddons.config.ConfigValues.loadValues(ConfigValues.java:243)
	at codes.biscuit.skyblockaddons.SkyblockAddons.postInit(SkyblockAddons.java:140)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:560)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:211)
	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:189)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:118)
	at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:742)
	... 10 more
```

**System Information**
 - OS: [ Windows 10]
 - Minecraft Version: [ 1.8.9]
 - Forge Version: [ 11.15.1.2318]
 - SkyblockAddons Version: [ 1.6 build 7]

**Other Installed Mods**
If you have any other mods installed, please list them here along with their version number.
- 0 Other mods

**Additional info**

**Comment by 116s** [comment]
Hello!

This bug is most probably fixed by #469.

**Comment by ILikePlayingGames** [comment]
Closing this issue since it's a duplicate of #467
